python/lsst/ctrl/bps/wms/panda/idds_tasks.py

```diff
@@ -225,3 +234,12 @@ def get_input_file(self, job_name):
         quantum graph file name
         """
         return next(iter(self.bps_workflow.nodes.get(job_name).get("inputs")))
+
+    def get_pseudo_input_file_name(self, jobname):
+        qgraph_node_ids = self.bps_workflow.nodes.get(jobname).get("job").qgraph_node_ids
+        if qgraph_node_ids:
+            pseudo_input_file_name = self.qgraph_file+"+"+jobname + "+" + qgraph_node_ids[0].buildId + \
+                "+" + str(qgraph_node_ids[0].number)

```

Comment:
We agreed this was fine for this ticket, but just pointing it out for future.  By limiting the node_ids to a single value, this code works today but definitely will not work as soon as a single GenericWorkflow job executes more than one Quantum.